src/routes/data/transformers.rs

```diff
@@ -76,6 +76,14 @@ impl TryFrom<storage::types::Locker> for super::types::RetrieveCardResponse {
     }
 }
 
+impl From<storage::types::Fingerprint> for super::types::FingerprintResponse {
+    fn from(value: storage::types::Fingerprint) -> Self {
+        Self {
+            card_fingerprint: value.card_fingerprint.expose(),
```

Comment:
No need to expose, `Secret` gets serialized properly, let's keep the type information until the very end